e2e/protractor.conf.js

```diff
@@ -33,11 +33,12 @@ exports.config = {
     },
     plugins: [{
         package: 'protractor-screenshoter-plugin',
-        screenshotPath: './REPORTS/e2e',
+        screenshotPath: './functional-output/e2e/',
```

Comment:
Shall we update the path in .gitignore to reflect this change?

Reply:
yep, we should